CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 0.18.2 - 2021-07-30
```

Comment:
This should be `0.18.1` - we just remove the `-alpha` part from the version for a patch release